core/process/data.loader.php

```diff
@@ -426,10 +426,10 @@
 		// get all mythic pokemon
 		$req 		= "SELECT DISTINCT pokemon_id, disappear_time FROM pokemon
 				   WHERE pokemon_id IN (".implode(",", $mythic_pokemons).")
-				   ORDER BY disappear_time DESC LIMIT 0,12";
+				   ORDER BY last_modified DESC LIMIT 0,12";
 	} else {
 		// get all pokemon
-		$req		= "SELECT DISTINCT pokemon_id, disappear_time FROM pokemon ORDER BY disappear_time DESC LIMIT 0,12";
+		$req		= "SELECT DISTINCT pokemon_id, disappear_time FROM pokemon ORDER BY last_modified DESC LIMIT 0,12";
```

Comment:
This is going to fail on newer mysql versions. See: http://piwigo.org/forum/viewtopic.php?pid=163884#p163884
You have to add last_modified to the SELECT DISTINCT part as well.